modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/dto/v1_0/converter/SXPBlueprintDTOConverter.java

```diff
@@ -156,12 +157,23 @@ private ElementInstance[] _translateElementInstances(
 
 		try {
 			for (ElementInstance elementInstance : elementInstances) {
-				SXPElement sxpElement = elementInstance.getSxpElement();
-
-				sxpElement.setTitle(
-					_language.get(locale, sxpElement.getTitle()));
-				sxpElement.setDescription(
-					_language.get(locale, sxpElement.getDescription()));
+				SXPElement elementInstanceSXPElement =
+					elementInstance.getSxpElement();
+
+				Long elementInstanceSXPElementId =
+					(Long)elementInstanceSXPElement.getId();
+
+				if (elementInstanceSXPElementId != null) {
```

Comment:
Is there a reason why elementInstanceSXPElementId would be a null? If not, I'd remove the null check and let it throw an exception and log it, in case of issues.

Reply:
This is for custom JSON elements since they don't have an ID.

Reply:
Right...forgot that.